### PR TITLE
Feature/translation issues#19

### PR DESCRIPTION
**Pull request by olsza**
Hay, adds a branch where you can add translations + display language flags, once configured.
Contains and does not include:
- [x] EN translation pattern
- [x] full Polish translation
- [x] Instruction how to add a new translation

ps. you have to remember to run ``` npm install && npm run prod ``` to download graphic flags
I didn't commit graphic flags and css styles in the public directory, because the above commands will do that

### CODE REVIEW

**Comment by olsza** [comment]
the end result, i.e. the view, can be seen here https://github.com/tighten/laravelversions/issues/19#issuecomment-776582522

File: resources/views/includes/langs.blade.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@if (Config::has('localized-routes.supported-locales'))
+    
+<ul class="max-w-6xl px-4 mx-auto sm:px-6 lg:px-8 -mt-9 float-right">
+    @foreach (Config::get('localized-routes.supported-locales', []) as $lang)
+        <li class="inline-block">
+            <a hreflang="{{ $lang }}" href="{{ \Route::localizedUrl($lang) }}" title="{{ $lang }}">
```
Comment (mattstauffer) [inline resources/views/includes/langs.blade.php:6]:
```suggestion
            <a hreflang="{{ $lang }}" href="{{ Route::localizedUrl($lang) }}" title="{{ $lang }}">
```

File: resources/views/includes/meta-langs.blade.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@foreach (Config::get('localized-routes.supported-locales', []) as $lang)
+    <link rel="alternate" hreflang="{{ $lang }}" href="{{ \Route::localizedUrl($lang) }}" />
```
Comment (mattstauffer) [inline resources/views/includes/meta-langs.blade.php:2]:
```suggestion
    <link rel="alternate" hreflang="{{ $lang }}" href="{{ Route::localizedUrl($lang) }}" />
```

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -4,9 +4,11 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        <title>{{ $title ? $title . ' - ' : '' }}Laravel Versions</title>
+        <title>{{ $title ? $title . ' - ' : '' }}{{ __('Laravel Version') }}</title>
```
Comment (mattstauffer) [inline resources/views/layouts/app.blade.php:7]:
```suggestion
        <title>{{ $title ? $title . ' - ' : '' }}{{ __('Laravel Versions') }}</title>
```

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -32,15 +34,22 @@
...
             </main>
 
             <div class="my-8 text-center">
-                Brought to you by the lovely folks at <a href="https://tighten.co/" class="text-blue-800 underline hover:text-blue-600">Tighten</a>.
+                {{ __('Brought to you by the lovely folks at') }} <a href="https://tighten.co/" class="text-blue-800 underline hover:text-blue-600">Tighten</a>.
+                <br>
+                <a href="https://github.com/tighten/laravelversions" class="text-blue-800 underline hover:text-blue-600">{{ __('Source on GitHub') }}</a> | <a href="/api/versions" class="text-blue-800 underline hover:text-blue-600">{{ __('Data available in JSON format') }}</a>
+                | <a href="https://twitter.com/laravelversions" class="text-blue-800 underline hover:text-blue-600">{{ __('Follow on Twitter for important dates') }}</a>
+
+                @if ( ! App::isLocale( Config::get('localized-routes.omit_url_prefix_for_locale') ) )
```
Comment (mattstauffer) [inline resources/views/layouts/app.blade.php:49]:
```suggestion
                @if (! App::isLocale(Config::get('localized-routes.omit_url_prefix_for_locale')))
```

**Review by mattstauffer — CHANGES_REQUESTED** [review]
Great work!

A few notes attached below.

And, I remember that with Onramp, it was hard to choose flags because some languages (e.g. English, Spanish) are attached to more than one country and so the flag isn't the best icon. Could you update that dropdown to instead use the language name, like we did on Onramp?

Thanks so much! Really great work here!

**Comment by olsza** [comment]
ok @mattstauffer, I know that.
Or maybe not to include, for example, in the configuration of language names, simply give the country code?
However, do you use the full names of the languages?

**Comment by mattstauffer** [comment]
image

^^ This is Onramp.

**Comment by mattstauffer** [comment]
But in the configuration, we keep it with just the code:

https://github.com/tighten/onramp/tree/main/resources/lang

**Comment by olsza** [comment]
new view of language menu:

Zrzut ekranu z 2021-02-16 00-10-58
Zrzut ekranu z 2021-02-16 00-06-16

**Comment by olsza** [comment]
I fixed style menu languages
FireFox on the left, to the right of Chrome

Zrzut ekranu z 2021-02-17 18-28-25

**Comment by mattstauffer** [comment]
Thanks for the update! I've been out of office with my family all week, but I'm taking a look now.

**Comment by mattstauffer** [comment]
Thanks for your hard work on this @olsza!

**Pull request merged**